jsx/Modal.tsx

```diff
@@ -1,17 +1,17 @@
-import {useState, PropsWithChildren, CSSProperties} from 'react';
+import {useState, PropsWithChildren, CSSProperties, ReactNode} from 'react';
 import Swal from 'sweetalert2';
 import Loader from './Loader';
 import {
   ButtonElement,
 } from 'jsx/Form';
 
-type ModalProps = PropsWithChildren<{
+export type ModalProps = PropsWithChildren<{
   throwWarning?: boolean;
   show: boolean;
   onClose: () => void;
   onSubmit?: () => Promise<any>;
   onSuccess?: (data: any) => void;
-  title?: string;
+  title?: ReactNode;
```

Comment:
changed because `modules/electrophysiology_browser/jsx/react-series-data-viewer/src/eeglab/EEGLabSeriesProvider.tsx` is passing an element to title. I think it's fine to accept ReactNode as title within reason.